pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -84,12 +85,29 @@ func (pc *PackageControllerClient) EnableCuratedPackages(ctx context.Context) er
 	if pc.managementClusterName != pc.clusterName {
 		return pc.InstallPBCResources(ctx)
 	}
-	ociUri := fmt.Sprintf("%s%s", "oci://", pc.uri)
-	registry := GetRegistry(pc.uri)
-
-	sourceRegistry := fmt.Sprintf("sourceRegistry=%s", registry)
+	ociURI := fmt.Sprintf("%s%s", "oci://", pc.registryMirror.ReplaceRegistry(pc.chart.Image()))
+	var values []string
 	clusterName := fmt.Sprintf("clusterName=%s", pc.clusterName)
-	values := []string{sourceRegistry, clusterName}
+	if pc.registryMirror != nil {
+		// account is added as part of registry name in package controller chart
```

Comment:
sorry I'm still missing to understand why this is necessary
why are we adding that to the `sourceRegistry` and `defaultRegistry`?

Reply:
https://github.com/aws/eks-anywhere-packages/blob/main/charts/eks-anywhere-packages/values.yaml#L15-L18
they incorporate account into the registry address

Reply:
Got you, can you add that info to the comment? I'm sure I'll read this code again in the future and forgot why this was done and that will help me :)